app/views/posts/index.html.erb

```diff
@@ -1,10 +1,39 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-  </head>
-
-  <body>
-    <h1>Here is a list of posts for a given user.</h1>
-  </body>
-</html>
+<div class="posts">
+
+  <div class="user_item">
+    <div class="user_img">
+      <img src=<%= @user.photo %> alt="user_img">
+    </div>
+    <div class="user_desc">
+      <h2 class="user_name"><%= @user.name %></h2>
+      <p class="user_post_number">Number of posts: <%= @user.posts_count %></p>
+    </div>
+  </div>
+
+  <ul class="post_list">
+
+    <% @posts.each do |post| %>
+
+      <li class="post_item_desc">
+
+        <div class="post_item">
+          <h4><%= post.title %></h4>
+          <p class="post_text"><%= post.text %></p>
+          <p class="comments_likes">
+            Comments: <%= post.comments_count %>, 
```

Comment:
- [ ] Please adjust your layout to match the given wireframe:

yours|given
---|---
![image](https://github.com/binyamolango/Blog-App/assets/91486791/2cf7db64-eb56-4210-a418-f0c9a95e139d)|![blog_user_all_posts](https://github.com/binyamolango/Blog-App/assets/91486791/e53e86e4-daeb-426c-8f3c-376b6d4d00e3)